app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    arguments_dictionary = {}
+
+    def inner(*args, **kwargs) -> None:
```

Comment:
Does this function return `None`?

Reply:
Arguments could be of any type, so I believe it should be Any. I've fixed the code